package.json

```diff
@@ -45,7 +45,7 @@
     "grunt-cli": "^0.1.13",
     "grunt-contrib-concat": "~0.3.0",
     "grunt-contrib-copy": "~0.5.0",
-    "grunt-contrib-jshint": "^0.10.0",
+    "grunt-contrib-jshint": "^1.0.0",
```

Comment:
Keeping it to test the task is fine. I was just thinking about JSHint, Testee and the release task.